node_modules/gh-users/tests/util.js

```diff
@@ -338,6 +341,63 @@ var assertGetUsersFails = module.exports.assertGetUsersFails = function(client,
 };
 
 /**
+ * Assert that the Terms and Conditions can be retrieved
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Function}           callback                        Standard callback function
+ * @param  {Object}             callback.termsAndConditions     The Terms and Conditions
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertGetTermsAndConditions = module.exports.assertGetTermsAndConditions = function(client, callback) {
+    client.user.getTermsAndConditions(function(err, termsAndConditions) {
+        assert.ok(!err);
+        assert.ok(termsAndConditions);
+        assert.ok(termsAndConditions.text);
+        assert.ok(termsAndConditions.lastUpdate);
```

Comment:
We can probably do more such as `_.isString(text)`, `_.isDate(lastUpdate)`, etc... maybe overkill to use the `generateTestFunction()` and extensions though?